tlatools/org.lamport.tlatools/javacc/tla+.jj

```diff
@@ -1629,26 +1631,13 @@ CompilationUnit() : {
   belchDEF();
 }{
   [ Prelude() ]
-  tempASTN = Module() { token_source.SwitchTo(0); }
-    /***********************************************************************
-    * In the classic practice of documentation by identifier name, we      *
-    * find that the definition of SwitchTo in                              *
-    * ConfigurationTokenManager.java is                                    *
-    *                                                                      *
-    *    static public void SwitchTo(int lexState)                         *
-    *                                                                      *
-    * so its argument seems to be a lexical state.  We also find in        *
-    * that file                                                            *
-    *                                                                      *
-    *    static int defaultLexState = 0;                                   *
-    *                                                                      *
-    * leading us to guess that this is setting the lexical state to        *
-    * DEFAULT.                                                             *
-    ***********************************************************************/
+  tempASTN = Module() {
+    // Reset tokenizer state to default value after parsing module.
+    token_source.SwitchTo(token_source.defaultLexState);
```

Comment:
`tla2sany.parser.TLAplusParserTokenManager.defaultLexState` is not final.  Still, passing `token_source.defaultLexState` instead of `0` constitutes no behavior change?